File: Daily Challanges/2026-07-July/2026-07-20.py
def is_golden_ratio(a, b):
    ratio = max(a, b) / min(a, b)
    return abs(ratio - 1.618) <= 0.01


print(is_golden_ratio(21, 34))
print(is_golden_ratio(15, 20))
print(is_golden_ratio(8, 13))
print(is_golden_ratio(10, 16))
print(is_golden_ratio(1618, 1000))
print(is_golden_ratio(88, 55))

File: Daily Challanges/2026-07-July/2026-07-20.js
function isGoldenRatio(a, b) {
  const ratio = Math.max(a, b) / Math.min(a, b);
  return Math.abs(ratio - 1.618) <= 0.01;
}

console.log(isGoldenRatio(21, 34));
console.log(isGoldenRatio(15, 20));
console.log(isGoldenRatio(8, 13));
console.log(isGoldenRatio(10, 16));
console.log(isGoldenRatio(1618, 1000));
console.log(isGoldenRatio(88, 55));
